src/framework/CMakeLists.txt

```diff
@@ -38,11 +38,23 @@ set(MPAS_FRAMEWORK_SOURCES
 
 add_library(framework ${MPAS_FRAMEWORK_SOURCES})
 set_MPAS_DEBUG_flag(framework)
-set(FRAMEWORK_COMPILE_DEFINITIONS
+if (MPAS_USE_SMIOL)
+    set(FRAMEWORK_COMPILE_DEFINITIONS
+        MPAS_SMIOL_SUPPORT
+        mpas=1
```

Comment:
Same as before, I think breaking out common defines would be good, leaving only the specifics within the conditional branch

Reply:
I moved the common defines out of this if/then clauses.